From 019f55a1daabb4b15632f3b49aceaa03a2b93c89 Mon Sep 17 00:00:00 2001
From: Uli Schlachter <psychon@znc.in>
Date: Sat, 22 Feb 2014 16:35:17 +0100
Subject: [PATCH] naughty: fix ldoc

lib/naughty.lua:65: ?: 'class' cannot have multiple values; {table,table}

Signed-off-by: Uli Schlachter <psychon@znc.in>
---
 lib/naughty.lua.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/naughty.lua.in b/lib/naughty.lua.in
index 6e77b5f..16dc384 100644
--- a/lib/naughty.lua.in
+++ b/lib/naughty.lua.in
@@ -61,7 +61,6 @@ naughty.config.notify_callback = nil
 -- @field low The preset for notifications with low urgency level
 -- @field normal The default preset for every notification without a preset that will also be used for normal urgency level
 -- @field critical The preset for notifications with a critical urgency level
--- @class table
 naughty.config.presets = {
     normal = {},
     low = {
-- 
1.9.0

